projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts

```diff
@@ -116,15 +121,30 @@ export class ResourceViewComponent implements OnInit, OnChanges, OnDestroy {
             });
     }
 
-    updateResource(newValue?: ReadValue, isDeletion?: boolean): void {
+    updateResource(value: BaseValue, isDeletion: boolean): void {
```

Comment:
a JsDoc comment should precede the method

Reply:
JsDoc added in 18faf9f